pkg/apis/options/providers.go

```diff
@@ -234,16 +243,59 @@ type LoginGovOptions struct {
 	PubJWKURL string `json:"pubjwkURL,omitempty"`
 }
 
+// Default sets sensible defaults for fields which have not been set from the
+// config
+func (oidcOptions *OIDCOptions) Default() {
+
+	if oidcOptions.IssuerURL == "" {
+		logger.Fatal("failed to set defaults for OIDC options, issuer URL has no sensible default")
```

Comment:
when `type == "google"`, there is a default for the `IssuerURL`. When it isn't, it very much depends. 

To be consistent, one could either drop the default-type google (creating a hassle for legacy users) or pass the `type` along the `*OIDCOptions`

What do you think?

Reply:
I think for this particular defaulting, we can handle it in the providers package rather than in the options, so for now I would remove this if block